### PR TITLE
add model lessons

### PR DESCRIPTION
**Pull request by Dmitryshl**

### CODE REVIEW

File: db/migrate/20220821175150_create_users.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ def change
       t.string :name
       t.string :email
       t.string :description
+
```
Comment (fooooxmr) [inline db/migrate/20220821175150_create_users.rb:7]: Remove this line from PR, please.

Comment (Terapool) [inline db/migrate/20220821175150_create_users.rb:7]: Extra line?

File: db/migrate/20220826101646_addreferenceslesson.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class Addreferenceslesson < ActiveRecord::Migration[7.0]
```
Comment (fooooxmr) [inline db/migrate/20220826101646_addreferenceslesson.rb:1]: Class name should be AddReferencesLesson

**Review by fooooxmr — APPROVED** [review]

File: db/migrate/20220826101646_addreferenceslesson.rb (hunk 2 above)
Comment (fooooxmr) [reply db/migrate/20220826101646_addreferenceslesson.rb:1]: Also file should be named: 20220826101646_add_references_lesson.rb

File: db/migrate/20220821175150_create_users.rb [hunk 3]
```diff
@@ -4,7 +4,6 @@ def change
       t.string :name
       t.string :email
       t.string :description
-
```
Comment (fooooxmr) [inline db/migrate/20220821175150_create_users.rb:7]: Extra line was removed. You should not have changes in this file. Please, add empty line :)

Comment (Dmitryshl) [reply db/migrate/20220821175150_create_users.rb:7]: ohh)

**Review by fooooxmr — APPROVED** [review]

**Review by Terapool — APPROVED** [review]

**Review by ritamma — APPROVED** [review]

**Review by evrusha — APPROVED** [review]

**Pull request merged**